lib/main.dart

```diff
@@ -31,13 +40,13 @@ class MyApp extends StatelessWidget {
         colorScheme: ColorScheme.fromSeed(seedColor: Colors.deepPurple),
         useMaterial3: true,
       ),
-      home: const MyHomePage(title: 'Flutter Demo Home Page'),
+      home: const MyHomePage(title: 'Subtitle Editor Demo Home Page'), //Прямо тут можно задать новое имя, передаётся в строке 40
     );
   }
 }
 
 class MyHomePage extends StatefulWidget {
-  const MyHomePage({super.key, required this.title});
+  const MyHomePage({super.key, required this.title}); //А здесь имя запрашивается, передаётся в строке 34
```

Comment:
Комментарии, ссылающиеся на определённый номер строки - то есть, если мы в будущем напишем код поверх этих строк, все подобные комментарии потребуют изменения. Не легче ли тогда ссылаться на имена классов и конструкторов?
```suggestion
  const MyHomePage({super.key, required this.title}); //А здесь имя запрашивается, передаётся в поле [MaterialApp.home]
```